pkg/client/apiutil/lazyrestmapper_test.go

```diff
@@ -401,4 +403,81 @@ func TestLazyRestMapperProvider(t *testing.T) {
 		g.Expect(err).To(gmg.HaveOccurred())
 		g.Expect(crt.GetRequestCount()).To(gmg.Equal(6))
 	})
+
+	t.Run("LazyRESTMapper can fetch CRDs if they were created in runtime", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("LazyRESTMapper can fetch CRDs if they were created at runtime", func(t *testing.T) {
```

The title actually confused me because I thought you mean't "Created in controller-runtime" :P